### DISCUSSION TITLE
Show spotlight & tooltip after scrolling

### DISCUSSION THREAD

**Discussion by AliNL** [gilbarbara/react-joyride#866] [Ideas]
Hi team,

We are using your product to make the tutorial of our website, it works perfectly except the scrolling issue.

We need it to auto scrolling between steps, and it works. But within a smooth scrolling, the position of target keeps changing, which makes the spotlight and tooltip blinking.

We'd like to have an option like `invisibleWhileAutoScrolling` to let them appear after scrolling. Or have a callback event of `autoScrollFinished`.

**Comment by AliNL** [comment]
Another reason is that the spotlight doesn't follow the target tightly (might because we have too much customized style, which slows down the rendering), so I wanna just hide it till the position settled.

**Comment by gilbarbara** [comment]
hey @AliNL 

This isn't something that we are willing to implement now.
You can try the controlled mode. Stop the tour when the next step needs to scroll, scroll the page and update the stepIndex and run it again.

**Reply by AliNL** [reply]
Ok, thanks for the suggestion. I will go with this workaround.
